graylog2-web-interface/src/routing/ApiRoutes.js

```diff
@@ -53,6 +53,9 @@ const ApiRoutes = {
     node: () => { return { url: '/system/cluster/node' }; },
     elasticsearchStats: () => { return { url: '/system/cluster/stats/elasticsearch' }; },
   },
+  ClusterConfigResource: {
```

Comment:
Not sure if these changes are relevant to the Customization Plugin system?

Reply:
These changes are needed in the enterprise PR, I could do them in an extra PR, but I thought it does not harm here.